posthog/models/action/util.py

```diff
@@ -98,7 +98,9 @@ def filter_event(
             conditions.append(f"{value_expr} LIKE %({prop_name})s")
             params.update({prop_name: f"%{step.url}%"})
 
-    conditions.append(f"event = %({prepend}_{index})s")
```

Comment:
Why not use just `None` as the "no event requirement" value? That'd align with handling for other fields like `step.url`